torchci/pages/benchmark/compilers.tsx

```diff
@@ -264,12 +262,13 @@ export default function Page() {
         <Typography fontSize={"2rem"} fontWeight={"bold"}>
           TorchInductor Performance DashBoard
         </Typography>
+        <div>Here</div>
```

Comment:
Is this a debug text?  It shows up on the page

Reply:
oh nice catch, yes! let me remove it!